.github/workflows/snap.yml

```
@@ -0,0 +1,68 @@
+name: snap
+
+on: [push]
```

Should we only push to snap from the master branch? Could do something like this?
```suggestion
on:
  push:
    branches:
      - master
```
Or we could deploy only when a release tag is pushed? (the same way continous deployment is currently configured)
```suggestion
on:
  push:
    tags:
      - "v*.*.*"

```

---

> Sorry for the delay.
> 
> Thank you for doing this! I've added the secrets.
> 
> Just one comment about whether we want to run the snap script on every push to every branch or only master or only when a release tag is created?

Good question. I would run on every commit that leads to a release. That way we are warned when changes break snap builds. The workflow only pushes on tags though.